### ISSUE TITLE
Want local-juno config

### ISSUE THREAD

**Issue by taiki1frsy** [confio/ts-relayer#241]
I'd like to have local-juno setup helper in ts-relayer for the test of the contract that enable ibc because it's major destination for CosmWasm contract.

**Comment by ethanfrey** [comment]
yup. I use wasmd as a placeholder for any generic cosmwasm chain.

If someone wants to add local juno into the ci and add some tests that use it, I am happy to merge that

**Comment by taiki1frsy** [comment]
At lease, I want to use that in these two repo:
https://github.com/taiki1frsy/simple-ibc-callback
https://github.com/taiki1frsy/multichain-contract-wallet-with-ica

**Comment by taiki1frsy** [comment]
@ethanfrey 
Are those insufficient?
If so, what 's line?
I can ask community in discord.
